### PR TITLE
Chore/allow toggling of realtime in table editor

### PR DESCRIPTION
**Pull request by joshenlim**
Add an option to toggle realtime for a table in the table editor side panel editor
- Available when creating/updating/duplicating a table
- Implementation is specifically for the supabase_realtime publication

image

### CODE REVIEW

File: studio/stores/pgmeta/MetaStore.ts [hunk 1]
```diff
@@ -512,6 +570,7 @@ export default class MetaStore implements IMetaStore {
     const columnIds = columns.map((column) => column.id)
 
     // Delete any removed columns
+    // @ts-ignore
```
Comment (alaister) [inline studio/stores/pgmeta/MetaStore.ts:573]: Why is this `@ts-ignore` needed?

**Review by alaister — APPROVED** [review]
Tested locally, and it works like a charm! A much better UX – I always forget to enable realtime.

Only one small nit:

**Comment by saltcod** [comment]
@joshenlim running into these errors when creating / duplicating a table.

I'm on this branch, and on `develop` for the api. Something I'm missing?


CleanShot 2022-06-20 at 15 26 03@2x


CleanShot 2022-06-20 at 15 22 02@2x

**Comment by joshenlim** [comment]
@saltcod ahh yeah, you'll need to rebuild your infra locally

**Pull request merged**